org.testeditor.web.backend.persistence/src/main/java/org/testeditor/web/backend/useractivity/UserActivityBroker.xtend

```diff
@@ -0,0 +1,39 @@
+package org.testeditor.web.backend.useractivity
+
+import java.util.HashMap
+import java.util.List
+import javax.inject.Singleton
+
+@Singleton
+class UserActivityBroker {
+
+	val userActivities = new HashMap<String, List<UserActivity>>()
+
+	def void updateUserActivities(String user, List<UserActivity> activities) {
+		userActivities.put(user, activities)
+	}
+
+	def Iterable<ElementActivity> getCollaboratorActivities(String excludedUser) {
+		val elementActivityMap = new HashMap<String, Iterable<UserActivityData>>()
+		userActivities.filter[user, activities|user !== excludedUser].forEach [ activeUser, activities |
```

Comment:
MINOR: first filter could be written as "filter[user, __|user !== excludedUser]" to indicate that the second parameter is not used